developer-docs-site/docs/tutorials/run-a-fullnode-on-gcp.md

```diff
@@ -68,6 +68,7 @@ You can deploy a public FullNode on GCP by using the Aptos fullnode Terraform mo
     zone          = "c"            # Specify the zone suffix
     project       = "gcp-fullnode" # Specify your GCP project name
     era           = 1              # bump era number to wipe the chain
+    image_tag     = "dev_5b525691" # Specify the docker image tag to use
```

Comment:
maybe specify a changing image tag like `devnet`?

Reply:
using same image tag is not so great some time, for example, terraform might think there's no change, thus no updates. it would be better to actually specify the image tag and change with every release. but it's up to the operators to decide.

Reply:
yea that's true. I was just thinking if most operators just copy-pasta, they will get image pull backoff when deploying with a potentially old image tag that we've pruned. I guess most folks running production grade stuff like this on GCP will be competent enough to replace the tag with something newer :/